src/services/sync/strategies/WebUntis/WebUntisSchoolyearSyncer.js

```diff
@@ -0,0 +1,622 @@
+const assert = require('assert');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['assert' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4061937)